onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1214,6 +1214,50 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
 
+constexpr const char* GemmaRotaryEmbedding_ver1_doc = R"DOC(
+GemmaRotaryEmbedding is the implementation of rotary positional embeddings (RoPE). It includes sin and cos in GemmaRotaryEmbedding forward and apply_rotary_pos_emb
```

Comment:
Write done the math formulation for this kernel. E.g.,

def gemma_rotary_embedding(x, y, z):
  w = Sin(x)
  z = Cast(y, to=?) # for fp16 mix-precision training. Other types are not supported.
  ...